contracts/payloads/IGP85/PayloadIGP85.sol

```diff
@@ -214,6 +205,80 @@ contract PayloadIGP85 is PayloadIGPMain {
         }
     }
 
+    // @notice Action 5: Set Launch Limits for USD0-USDC & fxUSD-USDC
+    function action5() internal isActionSkippable(5){
+        
+        {
+            address USD0_USDC_DEX = getDexAddress(23);
+            // USD0-USDC DEX
+            {
+                // USD0-USDC Dex
+                Dex memory DEX_USD0_USDC = Dex({
+                    dex: USD0_USDC_DEX,
+                    tokenA: USD0_ADDRESS,
+                    tokenB: USDC_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 10_000_000, // $10M
+                    baseBorrowLimitInUSD: 0, // $0
+                    maxBorrowLimitInUSD: 0 // $0
+                });
+                setDexLimits(DEX_USD0_USDC); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(USD0_USDC_DEX, TEAM_MULTISIG, false);
+            }
+
+            IFluidDex(USD0_USDC_DEX).updateMaxSupplyShares(
+                    7_500_000 * 1e18 // $15M
+            );
+        }
+
+        {
+            address fxUSD_USDC_DEX = getDexAddress(24);
+            // fxUSD-USDC DEX
+            {
+                // fxUSD-USDC Dex
+                Dex memory DEX_fxUSD_USDC = Dex({
+                    dex: fxUSD_USDC_DEX,
+                    tokenA: fxUSD_ADDRESS,
+                    tokenB: USDC_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 10_000_000, // $10M
+                    baseBorrowLimitInUSD: 0, // $0
+                    maxBorrowLimitInUSD: 0 // $0
+                });
+                setDexLimits(DEX_fxUSD_USDC); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(fxUSD_USDC_DEX, TEAM_MULTISIG, false);
+            }
+
+            IFluidDex(fxUSD_USDC_DEX).updateMaxSupplyShares(
+                    7_500_000 * 1e18 // $15M
+            );
+        }
+    }
+    // @notice Action 6: Set Rebalancers for USD0-USDC & fxUSD-USDC
+    function action6() internal isActionSkippable(6){
+        {
+            address fSL21_USD0_USDC = getDexAddress(23);
+
+            // set rebalancer at fSL21 to reserve contract proxy
+            IFTokenAdmin(fSL21_USD0_USDC).updateRebalancer(
```

Comment:
even though the interface of SmartLending and fToken matches here, it would be better to create a separate interface to be more clear in the code. Let's create ISmartLending interface with the updateRebalancer method please